### DISCUSSION TITLE
ArcticDB Backend

### DISCUSSION THREAD

**Discussion by christophlins** [bmoscon/cryptofeed#961] [Ideas]
Now that Man has released[ArcticDB]( https://github.com/man-group/ArcticDB), I would love to see an updated backend example with some book feed entries!

**Comment by bmoscon** [comment]
at some point it will happen :)

**Comment by DennyZen** [comment]
Same to me.. 
Can't realize - does new [ArcticDB](https://github.com/man-group/ArcticDB) have TickStore type? What about backend for it?
@bmoscon

**Reply by jamesmunro** [reply]
The different stores in the original Arctic have been replaced by a "Static" vs "Dynamic" schema choice.  If your tick-data has (or may have) changing fields then you want DynamicSchema.  https://docs.arcticdb.io/latest/#library-setup
